docs/references/ic-interface-spec.md

```diff
@@ -1549,7 +1549,8 @@ defaulting to `I = i32` if the canister declares no memory.
 
     ic0.env_var_name_size : (index: I) -> I;                                                          // *
     ic0.env_var_name_copy : (index: I, dst: I, offset: I, size: I) -> ();                             // *
-
+    ic0.env_var_name_exists : (name_src: I, name_size: I) -> I;                                       // *
```

Comment:
```suggestion
    ic0.env_var_name_exists : (name_src: I, name_size: I) -> i32;                                       // *
```
since the return type is a Boolean (not a pointer or variable size)